www/include/Administration/parameters/general/help.php

```diff
@@ -178,7 +178,8 @@
 );
 $help['openid_connect_redirect_url'] = dgettext(
     'help',
-    'Your OpenId Connect redirect url (this server).'
+    'Your OpenId Connect redirect url (this server, {$scheme}, {$hostname} and {$port} can be used for substitions,'
+    . ' default is {$scheme}://{$hostname}:{$port}/centreon/index.php if left empty).'
```

Comment:
. ' default is {$scheme}://{$hostname}:{$port}/your_centreon_path/index.php if left empty).'